data.bloodpac.org/manifest.json

```diff
@@ -22,11 +23,217 @@
     "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2020.09",
     "ambassador": "quay.io/datawire/ambassador:1.4.2",
     "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2020.09",
-    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2020.09"
+    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2020.09",
+    "ssjdispatcher": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ssjdispatcher:2020.09",
+    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2020.09"    
+  },
+  "indexd": {
+    "arborist": "true"
```

Comment:
FYI - I believe this has the effect of enabling "centralized auth".  I think to do that correctly we need to run a batch job to add `authz` fields to the indexd records, and ensure we have our resources setup in `user.yaml`, but we should get our hands on the documentation for that first ...

Reply:
Thanks! This PR fails again and again. Should I remove this part or is it ok?